invenio_files_rest/helpers.py

```diff
@@ -147,7 +146,7 @@ def send_stream(
         try:
             filenames = {"filename": filename.encode("latin-1")}
         except UnicodeEncodeError:
-            filenames = {"filename*": "UTF-8''%s" % url_quote(filename)}
+            filenames = {"filename*": "UTF-8''%s" % quote(filename)}
```

Comment:
suggestion: it would be good to align with https://github.com/zenodo/zenodo-rdm/blob/8b81afaeba88ba00230bfcf189f6b62b9ef8adb0/site/zenodo_rdm/files.py#L125C46-L125C51
https://www.rfc-editor.org/rfc/rfc5987

Reply:
thanks for the suggestion
I did align this PR with the code you linked